Extensions/xkit_preferences.js

```diff
@@ -74,12 +76,40 @@ XKit.extensions.xkit_preferences = new Object({
 			XKit.tools.add_css(mobile_control_panel, 'mobile_xkit_menu');
 		}
 
+		const react_add_button = async (button) => {
+			const account_label = await XKit.interface.translate("Account");
+			const menu_label = await XKit.interface.translate("Menu");
+
+			const insert = () => {
+				if (button.isConnected) return;
+				const header = document.querySelector('#base-container > div > div > header');
```

Comment:
is there a problem with making this just `#base-container header`? Using `> div > div >` just seems more fragile and more likely to break with innocuous changes to tumblr's code.

Reply:
Good point! Not particularly. In the extremely unlikely scenario that Tumblr implements a page without a header at the top, this would prevent the code from targeting the header of a post... but that would be pretty wild. Much more likely that a normal change breaks this, as you said.

All of the header elements are inside `#base-container`, so at that point this can just be `header` 👍